README.md

```diff
@@ -8,17 +8,16 @@ This repository contains the source code for the cloudflare worker of Serlo ([ht
 
 You can run tests with `yarn test`. We use [jest](https://jestjs.io/) and thus all [jest command line options](https://jestjs.io/docs/en/cli) can be used. It is also possible to run tests against other environments:
 
-- `TEST_ENVIRONMENT=dev yarn test` – Test against `serlo-development.dev`
-- `TEST_ENVIRONMENT=staging yarn test` – Test against `serlo-staging.dev`
-- `TEST_ENVIRONMENT=production yarn test` – Test against `serlo.org`
+- `yarn test:dev` – Test against `serlo-development.dev`
+- `yarn test:staging` – Test against `serlo-staging.dev`
+- `yarn test:production` – Test against `serlo.org`
 
 ### Write tests
 
 At [`__test__/__utils__`](./__tests__/__utils__) there are utility functions for writing tests:
 
 - [`fetch-helper.ts`](./__tests__/__utils__/fetch-helper.ts):
-  - `fetchTestingEnvironment()` - Fetch against the current testing environment. Use this function whenever possible.
-  - `fetchLocally()` – Fetch always against the local source code. Use this function when testing against a testing environment is not possible since you need to mock something.
+  - `fetchSerlo()` - Fetch against the current testing environment. Use this function whenever possible. By setting `{ environment: TestEnvironment.Locally }` you can always test against the local environment.
```

Comment:
Another small thing: The wording "Fetch against …" is something I don't really understand.